src/library/columncache.cpp

```diff
@@ -90,3 +93,100 @@ void ColumnCache::setColumns(const QStringList& columns) {
     m_columnSortByIndex.insert(m_columnIndexByEnum[COLUMN_PLAYLISTTRACKSTABLE_ARTIST], sortNoCase);
     m_columnSortByIndex.insert(m_columnIndexByEnum[COLUMN_PLAYLISTTRACKSTABLE_TITLE], sortNoCase);
 }
+
+void ColumnCache::slotSetKeySortOrder(double notation) {
```

Comment:
Ah, thank you for Explaining. The CO is 0 = DEFAULT at start-up and the Preferences are always writing a new value. That is also the reason for the new CUSTOM value. 
Is the DEFAULT enum used somewhere? I think the name is misleading and should be called VIRGIN. What do you think? 

Please add here a comment about it to avoid future questions.